EthOn.rdf

```diff
@@ -1904,18 +1904,18 @@
         </owl:annotatedTarget>
         <rdfs:comment>All create transactions are directed toward a Contract Account they create.</rdfs:comment>
     </owl:Axiom>
-
+    
 
 
     <!-- http://ethon.consensys.net/EthOnConcept -->
 
     <owl:Class rdf:about="http://ethon.consensys.net/EthOnConcept">
         <suggestedStringRepresentation xml:lang="en">EthonConcept</suggestedStringRepresentation>
-        <rdfs:comment xml:lang="en">Groups al EthOn concepts.</rdfs:comment>
+        <rdfs:comment xml:lang="en">The purpose of concepts in EthOn is to group a finite number of concrete instances of states and/or relationships between states under a single indefinitely bounded term.</rdfs:comment>
```

Comment:
Thanks @chronaeon I enjoyed reading this excursion. a common understanding of the foundations is really necessary. However, EthOn would not be the place to describe the foundational nature of concept. This is actually done in the OWL and RDF specification. Here is a summary: http://ontogenesis.knowledgeblog.org/514

Summing up - in EthOn Concept is used synonymous to Class.

@webthethird Yes, we should definitely talk about the difference between the simple and full definition of the terms. But let's choose a class that actually has instances in Ethereum, like e.g. `Account` (#121). `AccountConcept` is purely organizational and has no representation in Ethereum.